metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -333,15 +331,9 @@ def get_aspects_for_entity(
 
         result: Dict[str, Optional[Aspect]] = {}
         for aspect_type in aspect_types:
-            record_schema: RecordSchema = aspect_type.__getattribute__(
-                aspect_type, "RECORD_SCHEMA"
-            )
-            if not record_schema:
-                logger.warning(
-                    f"Failed to infer type name of the aspect from the aspect type class {aspect_type}. Continuing, but this will fail."
-                )
-            else:
-                aspect_type_name = record_schema.props["Aspect"]["name"]
+            record_schema = aspect_type.RECORD_SCHEMA
```

Comment:
why this change?

Reply:
Just general code cleanup not that RECORD_SCHEMA is part of the DictWrapper interface